grunt/config/browserify.js

```diff
@@ -62,6 +62,52 @@ function simpleBannerify(src) {
   );
 }
 
+// What is happening here???
+// I'm glad you asked. It became really to make our bundle splitting work.
```

Comment:
Too late! I pressed merge.

It became really "hard". Or maybe I wrote "annoying"? I'm going to leave the mystery in there for now :)